cpuid/src/brand_string.rs

```diff
@@ -340,6 +362,12 @@ mod tests {
                 "This function should not return another type of error"
             ),
         }
+
+        // Test BrandString::from_vendor_id()
+        let bstr = BrandString::from_vendor_id(VENDOR_ID_INTEL);
+        assert!(bstr.as_bytes().starts_with(BRAND_STRING_INTEL));
+        let bstr = BrandString::from_vendor_id(VENDOR_ID_AMD);
+        assert!(bstr.as_bytes().starts_with(BRAND_STRING_AMD));
```

Comment:
Let s add a test for this branch also: "For other CPUs, we'll just expose an empty string.".

Reply:
Done